packages/docs/docs/compliance/cfr11.md

```diff
@@ -0,0 +1,16 @@
+# CFR Part 11 Certification
+
+:::caution Note
+
+This section is under construction.
+
+:::
+
+The following tutorial walks through CFR Part 11 certification requirements and documentation of Medplum's conformance, where appropriate.
+
+## Materials and Usage
+
+| Resource Name  | Description  | Access  |
+|---|---|---|
```

Comment:
Minor nit: If you have the ESLint plugin in VS Code, then you should be able to run the formatter and make the tables look pretty.